docs/main/existing-rhdh.md

```diff
@@ -54,6 +54,8 @@ In 1.3, the Orchestrator infrastructure is installed using the Orchestrator Oper
         serviceNamespace: sonataflow-infra
       rhdhOperator:
         enabled: false
+      networkPolicy:
+        rhdhNamespace: custom-rhdh   
```

Comment:
Maybe should be something like:

rhdhNamespace: ${RHDH_NAMESPACE}    # Replace with namespace RHDH is deployed in